SignalServiceKit/src/Storage/YapDatabaseConnection+OWS.m

```diff
@@ -0,0 +1,138 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "YapDatabaseConnection+OWS.h"
+#import <YapDatabase/YapDatabaseTransaction.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation YapDatabaseConnection (OWS)
+
+- (id)objectForKey:(NSString *)key inCollection:(NSString *)collection
```

Comment:
Done. In fact, I went you one better and significantly cleaned up these methods - DRY'ing up code, adding asserts, etc.